### DISCUSSION TITLE
slow image generation on RTX 4050 6GB VRAM?

### DISCUSSION THREAD

**Discussion by yashunxz** [lllyasviel/Fooocus#2509] [Q&A] [answered]
I have RTX 4050 6gb VRAM and it still take alot of time to generate like 1.5min per image where is the problem and how can i fix it, plus while generating images in task manager i saw that only half of 6gb vram is been used ?, how can i make it use my GPU 100%? and does slow ssd has anything to do with the speed ?

**Comment by mashb1t** [comment, marked as answer]
Please check and identify which of your hardware is the bottleneck in task manager and if you have sufficient RAM / swap storage available.
If your drive is always used 100% it is impacting the performance, same for VRAM/RAM.
